external-import/misp/src/misp.py

```diff
@@ -329,7 +325,7 @@ def run(self):
                 import_from_date = datetime.fromisoformat(self.misp_import_from_date)
 
             # Prepare the query
-            kwargs = dict()
+            kwargs = {}
```

Comment:
No change needed here, it indicates the same thing

Reply:
Yes, except the first one raise a warning on pylint, but nevermind, I only need the fix.